src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -713,28 +736,20 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                     supportsFeatureService.SupportsCodeFixes(document))
                 {
                     var result = await Task.Run(
-                        () => provider._codeFixService.GetFirstDiagnosticWithFixAsync(
+                        () => provider._codeFixService.GetWorstFixableSeverity(
                             document, range.Span.ToTextSpan(), cancellationToken),
                             cancellationToken).ConfigureAwait(false);
 
-                    if (result.HasFix)
-                    {
-                        Logger.Log(FunctionId.SuggestedActions_HasSuggestedActionsAsync);
-                        return true;
-                    }
-
-                    if (result.PartialResult)
+                    if (result != null)
                     {
-                        // reset solution version number so that we can raise suggested action changed event
-                        Volatile.Write(ref _lastSolutionVersionReported, InvalidSolutionVersion);
```

Comment:
Looks like we need to track this, it lets the suggested action code determine if it is analyzing an up to date solution. Restored this code.